class/defaults.yml

```diff
@@ -1,6 +1,8 @@
 parameters:
   openshift4_monitoring:
     namespace: openshift-monitoring
+    # TODO: select based on reported OCP version once we have dynamic facts
+    manifests_version: release-4.7
```

Comment:
Is it possible that we can select this version from another parameter in the hierarchy?
Isn't there one that ultimately defines which release channel the cluster is using?
(yes I read the note about dynamic facts)
alternatively, we could set this parameter in the commodore defaults to reference another value via Reclass?

Reply:
We have `openshift4_version.spec.channel`, which is set to `stable-4.x` or `fast-4.x`, but we'd have to mangle that value to get to `release-4.x`, which is not possible in Reclass.

As far as I'm aware we don't have a parameter in the hierarchy which we can use to generate the value for this parameter at the moment.

Additionally, it's much easier when the component provides a somewhat sane default without external dependencies. We can always implement a "smarter" value for this parameter in our defaults repo, while having as few external dependencies as possible in the component itself.